arches/app/models/graph.py

```diff
@@ -1765,10 +1765,14 @@ def publish(self, user, notes=None):
                     language = models.Language.objects.get(code=language_tuple[0])
 
                     translation.activate(language=language_tuple[0])
+                    serialized_graph=JSONDeserializer().deserialize(JSONSerializer().serialize(self, force_recalculation=True))
```

Comment:
@robgaston - Was undecided whether to leave the sorting here or to push it into the serialize() function. Ended up leaving it here as putting it in serialize() seemed a bit too heavy. Moving would further ensure the order is correct everywhere. Let me know if you have a preference.